mos.yml

```diff
@@ -94,6 +94,7 @@ conds:
         - ["wifi.sta_ps_mode", "i", 0, {title: "Power save mode for station: 0 - none, 1 - min, 2 - max."}]
         - ["wifi.sta_all_chan_scan", "b", false, {title: "true to scan all channels or false to perform fast scan"}]
         - ["wifi.sta.protocol", "s", "BGN", {title: "802.11 Wi-Fi Protocol for STA Mode, defaults to BGN, can be any combination of BGNLR. Note LR only works between 2 ESP32 devices."}]
+        - ["wifi.sta.listen_interval", "i", 0, {title: "DTIM Listen Interval (e.g. 3 = 300ms)"}]
```

Comment:
can you call it listen_interval_ms and convert from milliseconds?

Reply:
Done